### DISCUSSION TITLE
recurring event: deleting effective only after logout/login

### DISCUSSION THREAD

**Discussion by Cer41Tom** [tutao/tutanota#6476] [Bugs]
When I delete one occurence of a repeating event (it is an all day event), the occurence stays visible until I log out and log in again.
This behavior applies to both Android and Windows apps.

Anybody else experiencing this?
Is this a bug?

**Comment by wrdhub** [comment]
Hey! Thanks for letting us know about this, I tested it and yeah it's a bug. I made an issue for it (#6491 ) and we will get it fixed!

**Comment by wrdhub** [comment]
Closing, follow issue for resolution : )
